infoscience_exports/exports/options_notices.py

```diff
@@ -168,8 +169,8 @@ def get_notices(options):
     notices = import_marc21xml(url, can_display_pending_publications)
 
     # check errors
-    if notices and 'error' in notices[0]:
-        options['error'] = notices[0]['error']
+    if notices and notices[0].get('message', '') != '':
+        options['error'] = notices[0]
```

Comment:
je vois que c'était déjà comme ça avant... mais c'est pas top la gestion des erreurs en harcodant un champ dans le premier résultat des notices... je verrai plutôt une exception lancée dans `import_marc21xml`, avec un traitement explicite ici.

Je rajouterais juste un commentaire `FIXME: use exception to manage errors`

et à améliorer dans une PR suivante